docs/src/routes/docs/[...4]wallets/[...36]finoaconnect/+page.md

```diff
@@ -0,0 +1,45 @@
+---
+title: FinoaConnect
+---
+
+## Wallet module for connecting FinoaConnect SDK to web3-onboard
+
+## Install
+
+<Tabs values={['yarn', 'npm']}>
+<TabPanel value="yarn">
+
+```sh copy
+yarn add @web3-onboard/core @web3-onboard/finoaconnect
+```
+
+  </TabPanel>
+  <TabPanel value="npm">
+
+```sh copy
+npm install @web3-onboard/core @web3-onboard/finoaconnect
+```
+
+  </TabPanel>
+</Tabs>
+
+## Usage
+
```

Comment:
```suggestion

Optional initialization object
```typescript
/** Optional object provided to the initiation of the wallet connector. 
 * When not included, the wallet connector service connects to FinoaConnect production systems.
 * @field {url} URL of the FinoaConnect backend systems to be used
 * @field {labelSuffix} arbitrary string label to denote the context of the URL field  */
export interface FinoaWalletOption {
  url?: string
  labelSuffix?: string
}
```

```

Reply:
You may need to fix the triple tick to close that code block since github doesnt like it in the suggestion but thats the general idea to ensure users know about the init object

Reply:
Sorry, I do not follow you here. The code segment (denoted with triple-ticks) is closed in line 45.
The  additional "typescript" word in the code section provides custom colouring on IDE's where recognition of typescript keywords are supported.

Reply:
I added the code changes